app/views/posts/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @author.name %></h2>
+<h2>Username</h2>
+<p>Number of posts: x</p>
+<% @author_posts.each do |post| %>
+  <a class="card-link" href="<%= user_post_path(post.id) %>">
+    <p>Post 1</p>
+    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vitae recusandae soluta tempore molestiae eum nam veniam autem. Mollitia tenetur ad provident, quisquam eius velit recusandae accusamus architecto hic, neque deserunt.</p>
+    <p>Comments: x Likes: x</p>
+    <p>Username: Comments 1</p>
+    <p>Username: Comments 2</p>
+    <p>Username: Comments 3</p>
+    <p>Username: Comments 4</p>
+    <p>Username: Comments 5</p>
+  </a>
+<% end %>
```

Comment:
- [ ] Kindly match the wireframe given for the `Posts#Index` view. You forgot include the image of the user for this page. 💻 🖱️ 

### Wireframe

![image](https://github.com/toyman640/blog/assets/77106962/6eb223be-1d7a-4e10-af12-92dbe7adef2b)

### Your Design

![image](https://github.com/toyman640/blog/assets/77106962/979a4420-3b72-4e1f-b15f-6a1e89ed2633)

Reply:
- [ ] [OPTIONAL] Kindly note that your design is yet to look exactly like the wireframe provided. The pagination button is missing, and the div holding the user image and the user name still needs to be fixed as stated by the first reviewer and the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="396" alt="Screenshot 2023-08-31 at 11 27 33" src="https://github.com/toyman640/blog/assets/58692590/d340e8f1-e232-42eb-8ba3-271b542cd645"> | ![blog_user_all_posts](https://github.com/toyman640/blog/assets/58692590/c3d0d20f-72f0-4796-b9f5-254ccc3ee904)